app/test_main.py

```diff
@@ -1,3 +1,124 @@
+import pytest
 from app.main import get_human_age
 
-# write your code here
+
+class TestResultAnimalAge:
+    @pytest.mark.parametrize(
+        "cat_age,dog_age,result",
+        [
+            (
+                0,
+                0,
+                [0, 0]
+            ),
+            (
+                14,
+                14,
+                [0, 0]
+            ),
```

Comment:
```suggestion
            (14, 14, [0, 0]),
```
And same for all cases bellow